### ISSUE TITLE
ReferenceError: clearTimeout is not defined

### ISSUE THREAD

**Issue by evoyy** [dlepaux/realtime-bpm-analyzer#49]
I'm trying to get this working with my React app and I am running into the following issue.

I copied `./node_modules/realtime-bpm-analyzer/dist/realtime-bpm-processor.js` into my public directory and followed the steps in the [Continuous Analysis strategy](https://www.realtime-bpm-analyzer.com/#continuous-analysis-strategy).

Upon activation, I'm seeing the following error repeating in the console:

````
realtime-bpm-processor.js:330 ReferenceError: clearTimeout is not defined
    at RealTimeBpmAnalyzer.<anonymous> (realtime-bpm-processor.js:260:11)
    at Generator.next (<anonymous>)
    at fulfilled (realtime-bpm-processor.js:7:26)
````

![Screenshot 2023-04-01 002226](https://user-images.githubusercontent.com/1623890/229243237-c70c07e8-23a1-4193-894a-119885a58e3a.png)

After learning about Audio Worklets, I think this might be a bug. If my understanding is correct, the processor runs in a separate execution context and does not have access to the DOM, therefore setTimeout/clearTimeout will not be available to the processor. Instead, the main thread has to be responsible for managing timers and triggering the stabilization by communicating with the worklet.

**Comment by dlepaux** [comment]
Hi @evoyy ! Thank you for the bug report. I've [patched](https://github.com/dlepaux/realtime-bpm-analyzer/commit/50f0a8b917581a8fc1262955773655929f0664d2) the issue and released the [version 3.1.5](https://github.com/dlepaux/realtime-bpm-analyzer/releases/tag/v3.1.5) that includes the bugfix.

**Comment by evoyy** [comment]
I confirm it is working. Thanks for the quick fix.
